app/labor/markdown_parser.rb

```diff
@@ -138,9 +138,12 @@ def escape_colon_emojis_in_codeblock(html)
   end
 
   def catch_xss_attempts(markdown)
-    bad_xss = ['src="data', "src='data", "src='&", 'src="&', "data:text/html"]
-    bad_xss.each do |xss_attempt|
-      raise ArgumentError, "Invalid markdown detected" if markdown.include?(xss_attempt)
+    bad_xss_regex = [
```

Comment:
The regular expressions don't change, since you are working on this PR already, woudl you mind pulling them out of this method and making them a constant?